db/schema.rb

```diff
@@ -575,6 +575,8 @@
     t.datetime "default_currency_confirmed_at"
     t.datetime "member_at"
     t.datetime "send_emails", default: -> { "CURRENT_TIMESTAMP" }
+    t.text "card_id"
```

Comment:
I don't think we need the card_id, this is what the `address` is.

Reply:
`card_id` is the equivalent of `card_id` found on `uphold_connection_for_channels`.`card_id`.